afl-fuzz.c

```diff
@@ -508,8 +533,13 @@ static void bind_to_free_cpu(void) {
   CPU_ZERO(&c);
   CPU_SET(i, &c);
 
+#ifdef __linux__
   if (sched_setaffinity(0, sizeof(c), &c))
     PFATAL("sched_setaffinity failed");
+#else
```

Comment:
would `#elif __FreeBSD__` make sense here?

Reply:
well ... I think only NetBSD has this call too in case someone wants to port it too ... but I ll change it